__tests__/__snapshots__/check-fixtures.js.snap

```diff
@@ -147,11 +147,70 @@ exports[`linting javascript.js fixture matches snapshot 1`] = `
     "ruleId": "no-undef",
     "severity": 2,
   },
+  {
+    "column": 121,
+    "endColumn": 121,
+    "endLine": 45,
+    "fix": {
+      "range": [
+        894,
+        894,
+      ],
+      "text": "
+",
+    },
+    "line": 45,
+    "message": "Insert \`⏎\`",
+    "messageId": "insert",
+    "nodeType": null,
+    "ruleId": "prettier/prettier",
+    "severity": 2,
+  },
 ]
 `;
 
 exports[`linting typescript.ts fixture matches snapshot 1`] = `
 [
+  {
+    "column": 48,
+    "endColumn": 61,
+    "endLine": 1,
+    "line": 1,
+    "message": "'three' is defined but never used.",
+    "messageId": "unusedVar",
+    "nodeType": "Identifier",
+    "ruleId": "@typescript-eslint/no-unused-vars",
+    "severity": 2,
+  },
+  {
+    "column": 55,
+    "endColumn": 61,
+    "endLine": 1,
+    "fix": {
+      "range": [
+        54,
+        60,
+      ],
+      "text": "string",
+    },
+    "line": 1,
+    "message": "Don't use \`String\` as a type. Use string instead",
+    "messageId": "bannedTypeMessage",
+    "nodeType": "Identifier",
+    "ruleId": "@typescript-eslint/ban-types",
+    "severity": 2,
+  },
+  {
+    "column": 1,
+    "endColumn": 14,
+    "endLine": 5,
+    "line": 5,
+    "message": "Do not use "@ts-ignore" because it alters compilation errors.",
+    "messageId": "tsDirectiveComment",
+    "nodeType": "Line",
+    "ruleId": "@typescript-eslint/ban-ts-comment",
+    "severity": 2,
+  },
```

Comment:
`@typescript-eslint/ban-ts-comment` added in 6.0: https://github.com/typescript-eslint/typescript-eslint/blob/v6.0.0/packages/eslint-plugin/src/configs/strict-type-checked.ts#L12